packages/sdk/src/sdk/api/emails/EmailsApi.ts

```diff
@@ -0,0 +1,188 @@
+import { base64 } from '@scure/base'
```

Comment:
created this file because we need an empty basePath for the `comms/pubkey/` file and `UserApi` has the default basepath of `v1`. lmk if you think there's a better path here

Reply:
Yeah unfortunately this is the only way to avoid that `v1` prefix unless we modified the template or something a little more involved...

Ideally this `pubkey` endpoint _would_ be on `v1` like everything else.

The other alternative would be bypassing `request` and calling `fetch` directly, but then you lose the middlewares and would have to select the DN manually. What you have I think is fine for now